README.md

```diff
@@ -0,0 +1,20 @@
+[![Coverage Status](https://coveralls.io/repos/github/praekelt/seed-auth-api/badge.svg?branch=develop)](https://coveralls.io/github/praekelt/seed-auth-api?branch=develop)
+[![Build Status](https://travis-ci.org/praekelt/seed-auth-api.svg?branch=develop)](https://travis-ci.org/praekelt/seed-auth-api)
+[![Requirements Status](https://requires.io/github/praekelt/seed-auth-api/requirements.svg?branch=feature%2Fissue-2-create-django-skeleton)](https://requires.io/github/praekelt/seed-auth-api/requirements/?branch=feature%2Fissue-2-create-django-skeleton)
+
+# seed-auth-api
+Seed Auth API. User and permissions store, authentication and authorization.
+
+## Running
+
+ * `pip install -e .`
+ * `psql -U postgres -c "CREATE DATABASE seed_auth;`
```

Comment:
Is that something that is done here? I thought the migrations might do it for us.. But I will have to try things and see what works. So I'm 👍  on adding it when we need it, so we can test if we actually need it.